components/Footer.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  Box,
+  chakra,
```

Comment:
nit: I think importing `Button` instead of `chakra` for `chakra.button` will reduce bundle size